CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
 ## Unreleased
+### General
+
+## 1.5.0 - 2022-03-14
+
+- Upgraded Otel dependencies to 1.9.0 and 0.28b0
```

Comment:
Nit: it seems the deps were upgraded to 1.10.0 and 0.29b0 with this change/release?